src/GitPull/source.extension.vsixmanifest

```diff
@@ -10,7 +10,7 @@
         <Tags>git</Tags>
     </Metadata>
     <Installation>
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0, 17.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.6, 17.0)" />
```

Comment:
It looks like `15.6` doesn't mean what I thought it meant. This version won't install on VS 2017 15.9. 😕 

@madskristensen What should this version be if I want it to install on 15.6 or above?

Reply:
15.6 is "15.0.27428". I use this website to find out https://docs.microsoft.com/en-us/visualstudio/install/visual-studio-build-numbers-and-release-dates?view=vs-2017